docs/developer-guide/get-started/emt-build-and-deploy.md

```diff
@@ -82,17 +86,101 @@ skip to [Create a bootable USB installer](#2-create-a-bootable-usb-installer).
 
 7. Build the toolkit image:
 
-   All available image configurations are stored in JSON files, located in the `imageconfigs` folder. If you want to configure any additional kernel command line arguments, follow the instructions in [Architecture Overview](../emt-architecture-overview.md#kernel-command-line).
-   For example, to build a RAW image without real-time extensions, use `edge-image.json` and run
-   the following command:
+   > **IMPORTANT**: Add the `REBUILD_PACKAGES=n` parameter only for the latest tags,
+   > as the build process uses prebuilt RPMs - downloads the latest available packages
+   > from the RPM repository.
 
    ```bash
    sudo make image -j8 REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/edge-image.json
    ```
 
-   The built image will be located in its separate directory under  `edge-microvisor-toolkit/out/images/`.
+### 1B. Build from a release tag
+
+1. Clone the release branch of edge-microvisor-toolkit repository:
+
+   Check the [tags](https://github.com/open-edge-platform/edge-microvisor-toolkit/tags) for
+   the `<release_tag_name>`. For example,
+   [3.0.20250718](https://github.com/open-edge-platform/edge-microvisor-toolkit/releases/tag/3.0.20250718).
+
+   ```bash
+   git clone https://github.com/open-edge-platform/edge-microvisor-toolkit --branch=3.0.20250718
+   ```
+
+2. Navigate to the `edge-microvisor-toolkit` directory:
+
+   ```bash
+   cd edge-microvisor-toolkit
+   ```
+
+3. Install prerequisites:
+
+   > **Note:** Requirements for building the toolchain were validated on Ubuntu 22.04 host.
+
+   ```bash
+   # Install required dependencies.
+   sudo ./toolkit/docs/building/prerequisites-ubuntu.sh
+
+   # Also supported:
+   #    make -C toolkit install-prereqs
+
+   # Fix go 1.23 link
+   sudo ln -vsf /usr/lib/go-1.23/bin/go /usr/bin/go
+   sudo ln -vsf /usr/lib/go-1.23/bin/gofmt /usr/bin/gofmt
+
+   # Install and configure Docker.
+   curl -fsSL https://get.docker.com -o get-docker.sh
+   sudo sh get-docker.sh
+   sudo usermod -aG docker $USER
+   ```
+
+   **You will need to log out and log back in** for user changes to take effect.
+
+4. Navigate to the `toolkit` subdirectory.
+
+   ```bash
+   cd ./toolkit
+   ```
+
+5. Rebuild the toolchain.
+
+   ```bash
+   sudo make toolchain \
+       PACKAGE_URL_LIST="" \
+       REPO_LIST="" \
+       DISABLE_UPSTREAM_REPOS=y \
+       REBUILD_TOOLCHAIN=y \
+       REBUILD_TOOLS=y
+   ```
+
+6. Customize the toolkit image.
+
+   You can include additional components to your toolkit image by adding RPM packages.
+   For more details, see
+   [Customize Your Edge Microvisor Toolkit Image](./emt-building-howto.md#customize-your-edge-microvisor-toolkit-image).
+
+7. Rebuild RPM packages.
+
+   ```bash
+   # Rebuild packages/RPMs local toolchain
+   sudo make build-packages REBUILD_TOOLS=y VALIDATE_TOOLCHAIN_GPG=n
+   ```
+
+7. Build the toolkit image:
+
+   > **IMPORTANT**: To create an image with an older tag, like
```

Comment:
should this be mention in the earlier steps ? because steps 5 and steps 7 here will take the longest time to build

Reply:
Moved with https://github.com/open-edge-platform/edge-microvisor-toolkit/pull/561/commits/e75b7dade1fc3c1162dcfeda5e87c412186e2a74